worksheets.go

```diff
@@ -137,6 +137,19 @@ func NewDefinitions(reader io.Reader, opts ...Options) (*Definitions, error) {
 					def.dependents[dependent.index] = append(def.dependents[dependent.index], field.index)
 				}
 			}
+			if field.constrainedBy != nil {
```

Comment:
Instead of adding this block (which is a copy paste of the handling of `computedBy`), why not have
```
fieldTrigger := field.computedBy
if fieldTrigger == nil {
  fieldTrigger = field. constrainedBy
}
```
then reuse the block above with `s/field.computedBy/fieldTrigger/`